app/services/reaction_handler.rb

```diff
@@ -146,13 +146,17 @@ def sink_articles(reaction)
   end
 
   def send_notifications(reaction)
+    return unless ReactionCategory.notifiable.include?(reaction.category.to_sym)
+
     Notification.send_reaction_notification(reaction, reaction.target_user)
     return unless reaction.reaction_on_organization_article?
 
     Notification.send_reaction_notification(reaction, reaction.reactable.organization)
   end
 
   def send_notifications_without_delay(reaction)
+    return unless ReactionCategory.notifiable.include?(reaction.category.to_sym)
```

Comment:
This is reaching into two dependencies in a relatively deep way — and twice in this class. What about adding some method like `Reaction#should_notify?` that encapsulates the logic?